src/main/java/com/shaft/tools/listeners/InvokedMethodListener.java

```diff
@@ -46,6 +46,11 @@ public void beforeInvocation(IInvokedMethod method, ITestResult testResult) {
                 } else {
                     ReportManager.logTestInformation(ReportManager.getTestClassName(), ReportManager.getTestMethodName(), "");
                 }
+                if (testMethod.getDescription() != null && testMethod.getDescription() != "") {
```

Comment:
Please apply codacy's suggestion below to use .equals() instead of == for string comparison